DetectorDescription/DDCMS/interface/DDCompactView.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DETECTOR_DESCRIPTION_DD_COMPACT_VIEW_H
+#define DETECTOR_DESCRIPTION_DD_COMPACT_VIEW_H
+
+// -*- C++ -*-
+//
+// Package:    DetectorDescription/Core
+// Class:      DDCompactView
+// 
+/**\class DDCompactView
+
+ Description: DD Compact View Facade
+
+ Implementation:
+     The DDCompactView facade serves as a launching point for a broader
+     refactor of monolithic or tightly-coupled systems in favor of more
+     loosely-coupled code.
+*/
+//
+// Original Author:  Ianna Osborne
+//         Created:  Wed, 22 May 2019 12:51:22 GMT
+//
+//
+
+namespace cms {
```

Comment:
Shouldn't a more specific namespace like `ddcms` be used, rather than the generic `cms`? I'd like to clear separation between the old and new `DDCompactView`.

Reply:
yes, we can change the namespace from cms to ddcms. We ought to do it consistently everywhere. BTW, the old DDCompactView is not in a namespace.

Reply:
Is this on the todo list for a future PR?